lab/metrics.md

```diff
@@ -62,11 +64,10 @@ http:
 
 The HTTP metrics provided by the metrics plugin include:
 
-- `rr_http_requests_queue` - Total number of queued requests waiting for a worker.
-- `rr_http_uptime_seconds` - Plugin uptime in seconds.
-- `rr_http_no_free_workers_total` - Total number of NoFreeWorkers errors.
```

Comment:
this metric does not exist

Reply:
It exists -> https://github.com/roadrunner-server/prometheus/blob/master/plugin.go#L58, but w/o `rr_http_`.

Reply:
Hmmm.... Sure.

Reply:
It exists within the `rr_http` namespace, w/o `rr_http_` is the mistake.

Reply:
So, no deletions from me